index.html

```diff
@@ -20,7 +20,7 @@ <h1 class="page-title">Contrast Ratio Calculator</h1>
               </div>
               <div class="input-items">
                 <div class="swatch-one"></div>
-                <input id="foreground-color" for="foreground-color" name="foreground-color" type="text" value="#000000" />
+                <input id="foreground-color" for="foreground-color" name="foreground-color" type="text" oninput="giveRatio()"/>
```

Comment:
For both of these inputs, it would cleaner to keep everything in the javascript file. 
Remove the `oninput` from here, and move it to the bottom of js file

```
backgroundColor.oninput = giveRatio;
foregroundColor.oninput = giveRatio;
```